data/device-tests/pixart-rf-GCBBTM25.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "MOUSE-GCBBTM25",
+  "interactive": false,
+  "steps": [
+    {
+      "url": "https://www.fwupd.org/downloads/c5b40f0e4a425c7332513e099a4fd8600feb33e1893b590156fafdbbfe8f0929-GCBBTM25_v0001.cab",
+      "components": [
+        {
+          "version": "0.0.1",
+          "guids": [
+            "657c927f-1889-596d-a0f1-d2abd9c067ab"
+          ]
+        }
+      ]
+    },
+    {
+      "url": "https://www.fwupd.org/downloads/b6bfa9c1f47252fb28d67bc3bf65bb9f11e2a3c1a32645450efdbdb64350c3c4-GCBBTM25_v0100.cab",
```

Comment:
This should be `emulation-url`, no?

Reply:
Due to some issues with the emulation data(https://github.com/fwupd/fwupd/pull/7330), it hasn't been uploaded yet.I will prepare the hardware to hughsie for debug.

So, there's only a link to the cab currently. Is the emulation URL necessary?

Reply:
Oh I see. Then yes this is sensible for now and once emulation data is fixed that URL can add later.

Thanks!

Reply:
Thank you!